pavelib/utils/test/suites/acceptance_suite.py

```diff
@@ -38,10 +38,11 @@ def cmd(self):
 
         cmd = (
             "DEFAULT_STORE={default_store} ./manage.py {system} --settings acceptance harvest --traceback "
-            "--debug-mode --verbosity {verbosity} {report_args} {extra_args}".format(
+            "--debug-mode --verbosity {verbosity} {pdb} {report_args} {extra_args}".format(
```

Comment:
I could change this code to build up a list of strings, and then join them together with `" ".join()`, which would ensure that there are no needless spaces. However, that would be a larger change, and you said that you didn't want to make the code even more complicated.